search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -331,6 +331,18 @@
                                     not-empty)]
      (str (string/join ". " citation-details) "."))))
 
+(defn- nil-if-not-provided
```

Comment:
Nitpick: This is a good candidate for `cmr.common.util` (included above as `util`). It's kind of the opposite of `inflate-nil-keys`

https://github.com/nasa/Common-Metadata-Repository/blob/master/common-lib/src/cmr/common/util.clj#L279